src/libs/collect.c

```diff
@@ -1193,6 +1196,8 @@ static void tree_view(dt_lib_collect_rule_t *dr)
                                 "   ON tagid = tag_id"
                                 " WHERE %s"
                                 " GROUP BY name,tag_id", where_ext);
+        // case insensitive for tags
+        string_compare = g_ascii_strcasecmp;
```

Comment:
what about multi-bytes utf8 characters ?

Reply:
> what about multi-bytes utf8 character

I'll look at this. This is new for me, if you have any tip or reference, this is welcome.

Reply:
I don't have solution but the documentation here make this clear I think:
https://developer.gnome.org/glib/stable/glib-String-Utility-Functions.html#g-ascii-strcasecmp

Reply:
Thanks. I've made sure that the comparison is made on utf8.